infra/containers.yaml

```diff
@@ -113,3 +113,25 @@ Resources:
               awslogs-region: us-west-2
               awslogs-stream-prefix: uatsrv
           Essential: true
+  autoscalingtarget:
+    Type: AWS::ApplicationAutoScaling::ScalableTarget
+    Properties:
+      MinCapacity: 2
+      MaxCapacity: 5
+      ResourceId: !Join ['/', [service, !Ref ecscluster, !GetAtt ecsservice.Name]]
+      ScalableDimension: ecs:service:DesiredCount
+      ServiceNamespace: ecs
+      RoleARN: arn:aws:iam::405662711265:role/aws-service-role/ecs.application-autoscaling.amazonaws.com/AWSServiceRoleForApplicationAutoScaling_ECSService
+  autoscalingpolicy:
+    Type: AWS::ApplicationAutoScaling::ScalingPolicy
+    Properties:
+      PolicyName: UATAutoScalingPolicy
```

Comment:
**Question** (non-blocking): What's the naming convention for these names? I've seen lower case (`uat`, `uatsrv`) as well as camelcase like this one.

Reply:
Haha, sorry!  Basically I try to use upper case if I think it's going to be in a UI, and lowercase if I think it's going to go in a log.  I have guessed wrong a few times - a cleanup is probably in order at the end of the week.